tests/scripts/all.sh

```diff
@@ -2128,6 +2128,7 @@ component_test_psa_crypto_config_accel_ecdsa_use_psa () {
 
     # All SHA-2 variants are needed for ECDSA signature tests,
     # but only SHA-256 is enabled by default, so enable the others.
+    scripts/config.py -f tests/include/test/drivers/config_test_driver.h set MBEDTLS_SHA1_C
```

Comment:
Note: the comment need updating, since SHA-1 is not part of SHA-2 :) So the comment should probably say "SHA-1 and all variants of SHA-2 are needed for ECDSA and X.509 tests".